selftests/unit/test_utils_process.py

```diff
@@ -23,6 +23,7 @@ def probe_binary(binary):
 
 
 TRUE_CMD = probe_binary('true')
+ECHO_CMD = probe_binary('echo')
```

Comment:
This is obviously in the wrong commit.

Reply:
ousp, sorry. I was adding this later. I'll fix these in next version.

Reply:
actually it's in the right commit as only with all the fixes this unittest can be added.